### ISSUE TITLE
Combat bug, very detrimental to players experience

### ISSUE THREAD

**Issue by itsRauzy** [Uuvana-Studios/longvinter-windows-client#1465]
When engaging in combat and coming out on the winning side of the battle, often your opponent vanishes out of thin air.  I am not sure what causes this as it happens in so many circumstances all over the map it leaves me baffled.  A buddy said people are just combat logging when they are losing to protect their stuff, but no I dont believe him.  I do not believe a game development team would allow players in a pvp survival situation to just get out of jail free whenever and wherever they would like to.  There is just simply no way a team would do that and completely ruin the experience for so many players who will not come back because of this.  So I told him he was wrong, and I came here to report this bug because there has to be a glitch in the system for this to be real....

**Comment by wevexOP** [comment]
Lock me into the game. 
![moneyman](https://user-images.githubusercontent.com/111082615/232198791-52244b54-2227-41ae-8b4c-e7024df0a923.png)

**Comment by wevexOP** [comment]
We are improving this game one comment at a time. I think they need to port the game to mobile because it feels like a mobile game.

![mad kirby](https://user-images.githubusercontent.com/111082615/232198882-b1f35667-2d47-46f6-a5e7-0e4f9572ccf3.jpg)

**Comment by wevexOP** [comment]
I agree with you, it's frustrating when opponents seem to vanish out of thin air after losing a battle. It can be hard to understand why this happens so frequently all over the map. I also don't think it's fair to assume that players are intentionally combat logging to protect their items when they're losing, as it would ruin the experience for others.

As you mentioned, it's unlikely that a game development team would allow players in a PvP survival situation to easily escape consequences. It's possible that there might be a glitch or bug in the system that's causing this issue. I appreciate you reporting it, as it's important for the game development team to be aware of such occurrences and take appropriate actions to ensure a balanced and enjoyable gaming experience for everyone.

I hope that the developers can look into this matter and find a solution. In the meantime, let's continue to enjoy the game and strive for fair and engaging battles!
